### PR TITLE
Tests for file download cards in message thread

### PR DESCRIPTION
**Pull request by prabhjot-msft**
# What
Tests for file download cards in message thread

# Why
https://skype.visualstudio.com/SPOOL/_workitems/edit/2783426

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-composites/tests/browser/chat/app/index.tsx [hunk 1]
```diff
@@ -53,7 +53,14 @@ function App(): JSX.Element {
   React.useEffect(() => {
```
Comment (alkwa-msft) [inline packages/react-composites/tests/browser/chat/app/index.tsx:53]: can we just import useEffect and avoid having to do React.useEffect?

File: packages/react-composites/tests/browser/chat/FileSharing.test.ts [hunk 2]
```diff
@@ -194,3 +194,39 @@ test.describe('Filesharing SendBox Errorbar', async () => {
...
+          {
+            name: 'SampleFile1.pdf',
+            extension: 'pdf',
+            url: 'https://sample.com/SampleFile.pdf',
+            uploadComplete: true
+          }
+        ])
+      })
+    );
+    // console.log(`xkcd: ${buildUrl(serverUrl, users[1])}`);
```
Comment (JamesBurnside) [inline packages/react-composites/tests/browser/chat/FileSharing.test.ts:221]: @prprabhu-ms this looks like one of your console logs 🧐

File: packages/react-composites/tests/browser/chat/FileSharing.test.ts [hunk 3]
```diff
@@ -194,3 +194,39 @@ test.describe('Filesharing SendBox Errorbar', async () => {
...
+    await waitForChatCompositeToLoad(page0);
+    await sendMessage(page0, testMessageText);
+    await waitForMessageDelivered(page0);
```
Comment (JamesBurnside) [inline packages/react-composites/tests/browser/chat/FileSharing.test.ts:224]: Looks like it's not waiting for the message to have been delivered:
image

Comment (JamesBurnside) [reply packages/react-composites/tests/browser/chat/FileSharing.test.ts:224]: `waitForMessageDelivered` looks poorly implemented and waits for _any_ messages to have been delivered

**Pull request closed without merging**